REST-client-in-cpp/RestClient.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#ifndef REST_CLIENT_HPP
+#define REST_CLIENT_HPP
+
+#include <iostream>
+#include <cstring>
```

Comment:
Check if all these include directives are needed in hpp file.
Most of these can be moved to the related .cpp file